### PR TITLE
Pr1

### PR DESCRIPTION
**Pull request by orthagonal**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -65,6 +66,15 @@ const runInterval = async(docker, options) => {
 };
 
 module.exports.start = (options) => {
+  log = Logr.createLogger({
```
Comment (jgallen23) [inline index.js]: any reason to not configure this on top of file?

File: index.js [hunk 2]
```diff
@@ -1,21 +1,28 @@
...
 
+let log = undefined; // use Logr for logging after it is configured
 const containers = {};
 
-// log when we go over threshold or when we go back under threshold:
-const logThresholdExceeded = (container, value, options) => {
+const logContainer = (container, value, options) => {
+  // verbose mode logs usage stats on every interval:
+  if (options.verbose) {
+    return log(['docker-ops', 'info'], `Container ${container.name} is using ${value}% of its CPU capacity`);
```
Comment (jgallen23) [inline index.js]: this isn't quite right - if verbose, we lose the warning and recovered tags

File: index.js [hunk 3]
```diff
@@ -1,21 +1,28 @@
...
+  }
+  // outside of verbose mode, only log if a container has exceeded its threshold for x intervals:
   const containerInfo = containers[container.id];
   if (value < options.cpuThreshold) {
     if (containerInfo.intervals > options.intervalsAllowed) {
-      console.log(`OKAY: Container ${container.name} CPU is now at ${value}%`);
+      log(['docker-ops', 'okay'], `Container ${container.name} CPU is now at ${value}%`);
```
Comment (jgallen23) [inline index.js]: please use same tag as ops lib

Comment (jgallen23) [inline index.js]: don't need docker-ops tag - that is implied

File: index.js [hunk 4]
```diff
@@ -65,6 +66,15 @@ const runInterval = async(docker, options) => {
 };
 
 module.exports.start = (options) => {
+  log = Logr.createLogger({
+    type: 'flat',
```
Comment (jgallen23) [inline index.js]: don't need type

**Review by jgallen23 — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (orthagonal) [reply index.js]: Later we might want to add CLI options that affect how the logger is configured, but I'll go ahead and move it to the top for now if there's no plans for that.

**Pull request merged**